Singer.API/Controllers/LegalGuardianUserController.cs

```diff
@@ -1,15 +1,46 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Singer.DTOs.Users;
+using Singer.Helpers.Exceptions;
 using Singer.Models.Users;
 using Singer.Services;
+using Singer.Services.Interfaces;
 
 namespace Singer.Controllers
 {
    [Route("api/[controller]")]
-   public class LegalGuardianUserController : DataControllerBase<LegalGuardianUser, LegalGuardianUserDTO, CreateLegalGuardianUserDTO>
+   public class LegalGuardianUserController : DataControllerBase<LegalGuardianUser, LegalGuardianUserDTO, CreateLegalGuardianUserDTO, UpdateLegalGuardianUserDTO>
    {
-      public LegalGuardianUserController(LegalGuardianUserService databaseService) : base(databaseService)
+      private ILegalGuardianUserService LegalGuardianUserService;
```

Comment:
This field (coz it's private) can be made `readonly` (I guess). Also, common naming conventions say private fields should be prefixed with `_`, so it would look like `_legalguardiansUserService`.